src/lib/background/domains/status/preset.js

```diff
@@ -9,14 +9,39 @@ const {
 const presets = require("../../../../data/domains");
 
 /**
- * Resolve flattr status for given URL or domain
+ * Retrieve only tree nodes which are relevant to the given host
+ * @param {string[]} hostParts - host parts (e.g. ["example", "com"])
+ * @return {any[]} tree nodes corresponding to given host parts
+ */
+function filterTree(hostParts)
+{
+  let treeNodes = [];
+  let tree = presets.status;
+  hostParts = hostParts.slice().reverse();
```

Comment:
nit seems like we don't need to create a new array and reverse it if we just changed the for-loop below to:

    for (let hostPart, i = hostParts.length - 1; hostPart = hostParts[i]; i--)

Reply:
Done. True, I tried to keep it simple because reverse loops are a bit more difficult to get right but given that this function is called quite often, the performance benefit is likely worth it.